src/services/school/logic/school.js

```diff
@@ -1,10 +1,13 @@
 const { URL } = require('url');
 const logger = require('../../../logger');
-const { SC_THEME, DOCUMENT_BASE_DIR } = require('../../../../config/globals');
+const { SC_THEME } = require('../../../../config/globals');
+const { Configuration } = require('@schul-cloud/commons');
```

Comment:
Codacy found an issue: [`@schul-cloud/commons` import should occur before import of `../../../logger`](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6045787)